src/main/webapp/app/oncokb-frontend-commons/src/components/RefComponent.tsx

```diff
@@ -12,14 +12,16 @@ export default class RefComponent extends React.Component<{
   componentType: 'tooltip' | 'linkout';
 }> {
   render() {
-    const parts = this.props.content.split(/pmid|nct|abstract/i);
+    const parts = this.props.content.split(/pmid|nct|abstract:/i);
```

Comment:
This will prevent abstract without colon to be recognized.

Reply:
Made a change, let me know what you think.